languages/__init__.py

```diff
@@ -12,17 +13,26 @@
 
 
 def execute(config, database, message, metadata: dict, data: io.BufferedReader):
-    lang = detect_language(metadata, data)
+    try:
+        lang = detect_language(metadata, data)
+        if not lang:
+            log.info("Could not detect language")
+            return
```

Comment:
Is this necessary, if `detect_language` raises exception when no language features are detected?